tools/cargo-each/src/check.rs

```diff
@@ -0,0 +1,91 @@
+use anyhow::{bail, Result};
+use cargo_metadata::Package;
+
+use crate::config::Config;
+
+pub(crate) fn configuration(package: &Package, config: &Config<'_>) -> Result<()> {
+    sets(package, config)?;
+
+    combinations(package, config)
+}
+
+fn sets(package: &Package, config: &Config<'_>) -> Result<()> {
```

Comment:
reading the function name I do not have any idea what it does

Reply:
While true, it's in the `check` module. If I named it `check_sets`, `clippy` complains when ran in pedantic mode.